Mathlib/GroupTheory/GroupAction/Basic.lean

```diff
@@ -244,14 +244,44 @@ theorem stabilizer_smul_eq_stabilizer_map_conj (g : G) (a : α) :
   rw [mem_stabilizer_iff, ← smul_left_cancel_iff g⁻¹, smul_smul, smul_smul, smul_smul,
     inv_mul_cancel, one_smul, ← mem_stabilizer_iff, Subgroup.mem_map_equiv, MulAut.conj_symm_apply]
 
+/-- The natural group equivalence between the stabilizers of two elements in the same orbit. -/
+def stabilizerEquivStabilizer {a b : α} {g : G} (hg : g • b = a) :
+    stabilizer G b ≃* stabilizer G a :=
+  ((MulAut.conj g).subgroupMap <| stabilizer G b).trans
+    (MulEquiv.subgroupCongr (by
+      simp [← hg, stabilizer_smul_eq_stabilizer_map_conj]))
+
+theorem stabilizerEquivStabilizer_apply {a b : α} {g : G} (hg : g • b = a) (x : stabilizer G b) :
+    stabilizerEquivStabilizer hg x = MulAut.conj g x := by
+  -- simp [stabilizerEquivStabilizer]
+  simp only [stabilizerEquivStabilizer, MulEquiv.trans_apply, MulEquiv.subgroupCongr_apply,
+    MulEquiv.coe_subgroupMap_apply, MulAut.conj_apply]
```

Comment:
I don't have a preference if you use the squeezed or unsqueezed one here, but I guess you don't want to keep the comment?